File: js/models/Person.js
// properties are directly passed to `create` method
function Person () {

	/* Attributes */
	name 			= '';
	email 			= '';
	password 		= '';
	typeOfAccount	= '';
	typeOfPerson 	= '';
	cnpj 			= '';
	cpf 			= '';
	gender 			= '';
	livesIn  		= {};
	worksIn 		= {};
	district 		= '';
	address 		= '';

	token 			= ''; //session information
}

/* GET/SET */
Person.prototype.getName = function(){
	return this.name;
};
Person.prototype.setName = function(name){
	this.name = name;
};
  	
Person.prototype.getEmail = function(){
	return this.email;
};
Person.prototype.setEmail = function(email){
	this.email = email;
};

Person.prototype.getPassword = function(){
	return this.password;
};
Person.prototype.setPassword = function(password){
	this.password = password;
};

Person.prototype.getTypeOfAccount = function(){
	return this.typeOfAccount;
};
Person.prototype.setTypeOfAccount = function(typeOfAccount){
	this.typeOfAccount = typeOfAccount;
};

Person.prototype.getTypeOfPerson = function(){
	return this.typeOfPerson;
};
Person.prototype.setTypeOfPerson = function(typeOfPerson){
	this.typeOfPerson = typeOfPerson;
};

Person.prototype.getCNPJ = function(){
	return this.cnpj;
};
Person.prototype.setCNPJ = function(cnpj){
	this.cnpj = cnpj;
};

Person.prototype.getCPF = function(){
	return this.cpf;
};
Person.prototype.setCPF = function(cpf){
	this.cpf = cpf;
};

Person.prototype.getLivesIn = function(){
	return this.livesIn;
};
Person.prototype.setLivesIn = function(livesIn){
	this.livesIn = livesIn;
};

Person.prototype.getWorksIn = function(){
	return this.worksIn;
};
Person.prototype.setWorksIn = function(worksIn){
	this.worksIn = worksIn;
};

Person.prototype.getDistrict = function(){
	return this.district;
};
Person.prototype.setDistrict = function(district){
	this.district = district;
};

Person.prototype.getAddress = function(){
	return this.address;
};
Person.prototype.setAddress = function(address){
	this.address = address;
};

/* Session token */
Person.prototype.getToken = function(){
	return this.token;
};
Person.prototype.setToken = function(token){
	this.token = token;
};

File: entrega-cd/app/back-end/webservice-java/src/br/edu/univas/si/tcc/trunp/model/Person.java
package br.edu.univas.si.tcc.trunp.model;

public class Person {

	private String name;
	private String email;
	private String password;
	private String typeOfAccount;
	private String typeOfPerson;
	private String district;
	private String address;
	private String photo;
	private City livesIn;
	private Company worksIn;

	
	
	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public String getPassword() {
		return password;
	}

	public void setPassword(String password) {
		this.password = password;
	}

	public String getTypeOfAccount() {
		return typeOfAccount;
	}

	public void setTypeOfAccount(String typeOfAccount) {
		this.typeOfAccount = typeOfAccount;
	}

	public String getTypeOfPerson() {
		return typeOfPerson;
	}

	public void setTypeOfPerson(String typeOfPerson) {
		this.typeOfPerson = typeOfPerson;
	}

	public String getDistrict() {
		return district;
	}

	public void setDistrict(String district) {
		this.district = district;
	}

	public String getAddress() {
		return address;
	}

	public void setAddress(String address) {
		this.address = address;
	}

	public String getPhoto() {
		return photo;
	}

	public void setPhoto(String photo) {
		this.photo = photo;
	}

	public City getLivesIn() {
		return livesIn;
	}

	public void setLivesIn(City livesIn) {
		this.livesIn = livesIn;
	}

	public Company getWorksIn() {
		return worksIn;
	}

	public void setWorksIn(Company worksIn) {
		this.worksIn = worksIn;
	}

}
